earth_enterprise/src/fusion/autoingest/sysman/khResourceManager.cpp

```diff
@@ -408,14 +415,14 @@ khResourceManager::TryActivate(void) throw()
       taskWaitingQueue.size() &&
       providers.size()) {
     // figure out which providers have CPU resources to spare
-    Providers availProviders;
```

Comment:
I don't think this line should be deleted. I'm getting a compile error that availProviders was not declared.

Reply:
As a separate issue, travis is not catching the error. The build fails but travis marks it as a success.

Reply:
Has this happened elsewhere vis a vis Travis CI giving false positives?